cmd/containerd/main.go

```diff
@@ -59,11 +55,6 @@ high performance container runtime
 			Value: "shim",
 		},
 		cli.StringFlag{
-			Name:  "socket, s",
-			Usage: "socket path for containerd's GRPC server",
-			Value: "/run/containerd/containerd.sock",
-		},
-		cli.StringFlag{
 			Name:  "metrics-address, m",
 			Usage: "tcp address to serve metrics on",
 			Value: "127.0.0.1:7897",
```

Comment:
Probably this should be PlatformFlags as well? (i.e. UNIX socket on UNIX in future)